.travis.yml

```diff
@@ -21,22 +22,23 @@ before_install:
 install:
   - pip freeze -l
   # Preinstalled travis pytest is not tracked by us
-  - pip uninstall pytest -y
+  # Preinstalled travis pbr is not tracked by us
```

Comment:
Instead of uninstalling Python packages that overlap between what Travis includes and what we use, it would better to simply run `pip install -r testinfra/requirements.txt` with the `-U` flag. This is preferable because (i) if the Python packages in Travis are updated, we don't have to worry about additional overlaps in the future and (ii) if they happen to be on the same version of an overlapping package, we don't have to reinstall it.

Last time I checked `pip install -r reqs.txt` will not upgrade or downgrade an already installed package even if requirements file specifies a different version of that package, whereas `pip install -U -r reqs.txt` will upgrade/downgrade that package to the version specified in the requirements file.

You can get rid of the three `pip freeze -l` commands I see in this diff too then, which look like they are used for debugging. Ensuring the right dependencies are present in Travis should always be confirmed by testinfra.